### DISCUSSION TITLE
Is it possible to show errors last?

### DISCUSSION THREAD

**Discussion by iskandarovBakshi** [eslint/eslint#17206] [General] [answered]
Hi Guys in my project I have lots of warnings when I run `eslint` but when I add new rule and run `eslint` output shows mixed warning/errors 

my question is is it possible to show errors last instead mixing with warnings?

**Comment by mdjermanovic** [comment, marked as answer]
Hi @iskandarovBakshi!

> my question is is it possible to show errors last instead mixing with warnings?

Yes, you can make a custom formatter that sorts the results as you want.

https://eslint.org/docs/latest/extend/custom-formatters

**Comment by aladdin-add** [comment]
yes, another choice is to use the `--quiet` option (if you only care about errors).

https://eslint.org/docs/latest/use/command-line-interface#--quiet
